src/main/java/org/gridsuite/shortcircuit/server/entities/FaultResultEntity.java

```diff
@@ -88,4 +120,8 @@ public boolean equals(Object o) {
     public int hashCode() {
         return getClass().hashCode();
     }
+
+    public double getPositiveMagnitude() {
+        return this.getFortescueCurrent() != null ? this.getFortescueCurrent().getPositiveMagnitude() : Double.NaN;
```

Comment:
Is it normal to only access 'fortescueCurrent' and never 'fortescueVoltage' ?

Reply:
Same as above, It was asked to store all retrieved data but it not everything is used for the time being